### DISCUSSION TITLE
Annotate outside collaborators in Organization

### DISCUSSION THREAD

**Discussion by gagf01** [community/community#40954] [Repositories]
Hi,

I don't know if I'm in the right category since there is no "Organization" category.

Our organization is starting to have a lot of outside collaborators. Pretty much all of them have funny internet names and we keep losing trace of who is who... It would be very nice if we could annotate our outside collaborators in the Organization -> People -> Outside collaborators view  to help identify them.

Thanks

**Comment by ossanna16** [comment]
Hi there @gagf01 and welcome to our community! Thank you for the feedback!

**Reply by chrheg** [reply]
This would be a really good feature to have.
